### PR TITLE
docs: add docs for actions

### PR DESCRIPTION
**Pull request by eliobischof**
closes: #3217

### CODE REVIEW

File: docs/docs/guides/customization/behavior.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+Now, make the action hook into the [external authentication flow](../../apis/actions#external-authentication-flow).
+
+1. In the **Flows <i class="las la-exchange-alt"></i>** section, select the **+ New** button.
+1. Select the **Flow Type** *External Authentication*.
+1. Select the **Trigger Type** *Post Creation*.
+1. In the **Actions** dropdown, check *addGrant*.
+1. Select the **Save** button.
+
+<!-- TODO: ## Test if your action works -->
```
Comment (eliobischof) [inline docs/docs/guides/customization/behavior.md:51]: I think we should wait for v2 to complete this

File: docs/docs/concepts/features/actions.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+Using the actions feature, *ORG_OWNERs* create a flow for each supported flow type.
+Each flow type provides its own events.
+You can hook into these events by assigning them an action.
+An action is composed of
+* a name,
+* a custom JavaScript code snippet,
+* an execution timeout in seconds,
+* a switch that defines if its corresponding flow should fail if the action fails.
+
+Within the JavaScript code, you can read and manipulate state.
```
Comment (livio-a) [inline docs/docs/concepts/features/actions.md:33]:
```suggestion
Within the JavaScript code, you can read and manipulate the state.
```

Comment (livio-a) [reply docs/docs/concepts/features/actions.md:33]: ?

File: docs/docs/guides/customization/behavior.md (hunk 1 above)
Comment (livio-a) [reply docs/docs/guides/customization/behavior.md:51]: fine by me

File: docs/docs/apis/actions.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+- `api.setPhone(string)`
+- `api.setPhoneVerified(bool)`
+- `api.metadata array<Metadata>`  
+  Push entries.  
+- `api.userGrants array<UserGrant>`  
+  Push entries.  
+  This field is only available for the post creation trigger
+
+
+### External authentication flow types <!-- TODO: Are these types correct? -->
```
[the remark this answers is not in the document]
Comment (livio-a) [reply docs/docs/apis/actions.md:79]: LGTM

**Review by fforootd — APPROVED** [review]

**Pull request merged**